### PR TITLE
Document how to authenticate to Google Cloud Storage

### PR DESCRIPTION
**Pull request by jean-philippe-martin**
Via a VM, your account, or a service account.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,35 @@ If you are looking for the codebase of the current production version of GATK, p
   ./gatk-launch PrintReadsSpark -I input.bam -O output.bam
   ```
 
+####Running GATK4 with inputs on Google Cloud Storage:
+
+* GATK can read BAM or VCF inputs from a Google Cloud Storage bucket. Just use the "gs://" prefix:
+  ```
+  ./gatk-launch PrintReads -I gs://mybucket/path/to/my.bam -L 1:10000-20000 -O output.bam
+  ```
+* For public datasets, that will just work. For private datasets you must set up authentication.
```
Comment (lbergelson) [inline README.md]: Is this true?  Who gets charged for the data transfer in this case?  The owner of the data?

File: README.md [hunk 2]
```diff
@@ -110,6 +110,35 @@ If you are looking for the codebase of the current production version of GATK, p
...
+* Option (a): run in a Google Cloud Engine VM
+  * If you are running in a Google VM then your credentials are already in the VM and will be picked up by GATK, you don't need to do anything special.
+* Option (b): use your own account
+  * Install [Google Cloud SDK](https://cloud.google.com/sdk/)
+  * Log into you account:
```
Comment (droazen) [inline README.md]: you -> your

File: README.md [hunk 3]
```diff
@@ -110,6 +110,35 @@ If you are looking for the codebase of the current production version of GATK, p
...
+  ```
+  gcloud auth application-default login
```
Comment (droazen) [inline README.md]: If you type `gcloud auth login` (without the 'application-default`), will it still work?

File: README.md (hunk 1 above)
Comment (droazen) [inline README.md]: Is this true? For public datasets you don't need to be authenticated at all?

**Comment by droazen** [comment]
Back to @jean-philippe-martin , merge after addressing comments

File: README.md (hunk 2 above)
Comment (jean-philippe-martin) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (jean-philippe-martin) [inline README.md]: no, it won't work.

File: README.md (hunk 1 above)
Comment (jean-philippe-martin) [inline README.md]: ah you may be right, let me double-check.

Comment (jean-philippe-martin) [inline README.md]: I was wrong. Technically the user doesn't need to be authenticated if you have an API key, but we aren't using one so the user needs to authenticate.

**Pull request merged**